app/controllers/merchants_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MerchantsController < ApplicationController
+  def index
+    @merchants = MerchantFacade.merchants_info
+  end
+
+  def show
+    @merchant = MerchantFacade.merchant_info(params[:id])
+    @items = MerchantFacade.item_info(params[:id])
```

Comment:
(low priority)
This line took me a moment to wrap my head around, since we're getting a collection of items back but still need to pass a merchant_id. I wonder if the `item_info` method could accept a keyword argument, so this line becomes something like `@items = MerchantFacade.item_info(merchant_id: params[:id])`